code/modules/clothing/suits/armor.dm

```diff
@@ -297,6 +297,10 @@
 /obj/item/clothing/suit/armor/riot/Initialize(mapload)
 	. = ..()
 	AddComponent(/datum/component/adjust_fishing_difficulty, 5)
+	init_rustle_component()
+
```

Comment:
why make this a proc if it's never meaningfully overrided?

Reply:
it's overriden on knight armor

Reply:
for future please do not mark questions like this resolved, I need to like, read your response

Reply:
ah I see wrong one

Reply:
I did that one for consistency even tho nothing really needs itx should I just do add_component here instead?